lib/src/components/layout/tree.dart

```diff
@@ -1069,8 +1069,10 @@ class TreeItemView extends StatefulWidget {
   final VoidCallback? onPressed;
   final VoidCallback? onDoublePressed;
   final ValueChanged<bool>? onExpand;
+  final ValueChanged<bool>? onHover;
   final bool? expandable;
   final FocusNode? focusNode;
+  final int? branchLineWidth;
```

Comment:
Maybe change this to `int` (non-nullable) is better, as there is already a default value of 16 and null would cause trouble on line 1184

Reply:
Its assigned in the constructor with the default value which is 16.